src/main/java/org/prebid/server/handler/StatusHandler.java

```diff
@@ -27,6 +28,7 @@ public StatusHandler(List<HealthChecker> healthCheckers, JacksonMapper mapper) {
 
     @Override
     public void handle(RoutingContext routingContext) {
+        routingContext.response().putHeader(HttpUtil.CONTENT_TYPE_HEADER, HttpHeaderValues.APPLICATION_JSON);
```

Comment:
This can be done in lambda below.

Reply:
idk why, but having this done in lambda crashes test